### ISSUE TITLE
one-shot results are wrong

### ISSUE THREAD

**Issue by saxg** [imatge-upc/rvos#19]
I test the code of one shot and the results of each clip are the same as the mask of the ground truth of the first frame. Maybe there is some trouble in the code of eval_previous_mask.py

**Comment by carlesventura** [comment]
This is very strange. Have you used the one-shot pre-trained model or have you trained the model for your own dataset? 

Best regards,

Carles
